src/Request.php

```diff
@@ -28,6 +28,11 @@ class Request
     const METHOD_TRACE   = 'TRACE';
     const METHOD_CONNECT = 'CONNECT';
 
+    /**
+     * @var string
+     */
+    private $raw = '';
```

Comment:
```suggestion
    private $rawPayload = '';
```

For consistency with the method name and more context.